submissions/kyldvs/src/SithDataManager.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @flow
+ */
+
+'use strict';
+
+import Dispatcher from './Dispatcher';
+import Sith from './Sith';
+import xhr from 'xhr';
+
+const URL = 'http://localhost:3000/dark-jedis/';
+
+// Keep track of requests so we can abort them later.
+let counter = 1;
+const requests = new Map();
```

Comment:
Do you actually need this `Map`? I was under the impression that the requests were supposed to be sequential, therefore I thought that there couldn't be more than one active request happening at the same time. If that's the case you just need to store the current active request (if there is one), then you wouldn't need to pass the id parameter into the `abort` function and you would only need to call the `abort` function once from the "scroll-down/up" actions.

Apart from that, thanks a lot for this contribution! I found it very instructive!